deployment/hasura/migrations/Aerie/14_actions/up.sql

```diff
@@ -0,0 +1,162 @@
+comment on column actions.action_definition.name is e''
+  'The name of the action.';
+comment on column actions.action_definition.description is e''
+  'The description of the action.';
+comment on column actions.action_definition.parameter_schema is e''
+  'The JSON schema representing the action''s parameters.';
+comment on column actions.action_definition.settings_schema is e''
+  'The JSON schema representing the action''s settings.';
+comment on column actions.action_definition.action_file_id is e''
+  'The ID of the uploaded action file.';
+comment on column actions.action_definition.workspace_id is e''
+  'The ID of the workspace the action is part of.';
+
+create trigger set_timestamp
+  before update on actions.action_definition
+  for each row
+  execute function util_functions.set_updated_at();
+
+create function actions.notify_action_definition_inserted()
+  returns trigger
+  security definer
+  language plpgsql as $$
+begin
+  perform (
+    with payload(action_definition_id,
+                 action_file_path) as
+           (
+             select NEW.id,
+                    uf.name
+             from merlin.uploaded_file uf
+             where uf.id = NEW.action_file_id
+           )
+    select pg_notify('action_definition_inserted', json_strip_nulls(row_to_json(payload))::text)
+    from payload
+  );
+  return null;
+end$$;
+
+create trigger notify_action_definition_inserted
+  after insert on actions.action_definition
+  for each row
+execute function actions.notify_action_definition_inserted();
+
+create table actions.action_run (
+  id integer generated always as identity,
+
+  settings jsonb not null,
+  parameters jsonb not null,
+  logs text,
+  error jsonb,
+  results jsonb,
+  status actions.action_run_status not null default 'pending',
+
+  action_definition_id integer not null,
+
+  created_by text,
+  created_at timestamptz not null default now(),
+
+  constraint action_run_synthetic_key
+    primary key (id),
+  foreign key (action_definition_id)
+    references actions.action_definition (id)
+    on delete set null,
+  foreign key (created_by)
+    references permissions.users
+    on update cascade
+    on delete set null
+);
+
+comment on table actions.action_run is e''
+  'The record of a single run of an action.';
+comment on column actions.action_run.settings is e''
+  'The supplied settings for the run of the action.';
+comment on column actions.action_run.parameters is e''
+  'The supplied parameters for the run of the action.';
+comment on column actions.action_run.logs is e''
+  'The logs produced by the action run.';
+comment on column actions.action_run.error is e''
+  'The error produced by the action run.';
+comment on column actions.action_run.results is e''
+  'The results produced by the action run.';
+comment on column actions.action_run.status is e''
+  'The status of the action run. pending -> in-progress -> failed or complete';
```

Comment:
I'd recommend removing the part after the first sentence.
```suggestion
  'The status of the action run.
```